news/2024/2024-06-27-osu-catch-world-cup-2024-finals-recap.md

```diff
@@ -0,0 +1,98 @@
+
+To make matters worse for the Chilean team, Poland was able to break them a second time. This time, [Phob]() led his team to a 5-2 advantage on [NM4](https://osu.ppy.sh/beatmapsets/2198872#fruits/4653265). Backs against the wall, Chile broke back immediately with a very solid Hidden score coming out from rookie [Thaemine](https://osu.ppy.sh/users/32442896) on [MX3](https://osu.ppy.sh/beatmapsets/2198806#fruits/4653125). On the very next map, [HR3](https://osu.ppy.sh/beatmapsets/2198812#fruits/4653138), [Thaemine](https://osu.ppy.sh/users/32442896) produced a brilliant full-combo cutting down Polands lead to only a single point.
+
+Both teams won their next picks: Poland won [HD2](https://osu.ppy.sh/beatmapsets/2198958#fruits/4653546) convincingly and Chile won [HR1](https://osu.ppy.sh/beatmapsets/2198791#fruits/4653099) on the back of an impressive full-combo from [Aryssiel](https://osu.ppy.sh/users/2389481). Poland is leading 6-5 and have the final pick to their disposal as well. There aren't many potential maps left in the pool and they make the wrong choice going for [NM3](https://osu.ppy.sh/beatmapsets/2198988#fruits/4653607). Chile secures a 6th point and give us the tiebreaker this match deserves! 
+
+The pressure on both teams is immense as a top 4 finish is on the line here. However, there is one man that steps up to the task. [trig0n](https://osu.ppy.sh/users/3704228), the number one in Poland, lives up to his name and absolutely outperforms the entire lobby with a 3-miss run on the [TB](https://osu.ppy.sh/beatmapsets/2198809#fruits/4653132). His astonishing performance helps secure Poland their first top 4 finish on the osu!catch world cup stage.
+
+### ::{ flag=CA }:: Canada vs. ::{ flag=PL }:: Poland
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2179927104&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+Canada convincingly won their match against Finland and also secured themselves top 4. As a reward, they get to battle it out with Poland which for both teams would be their first podium finish ever.
+
+This match started very similarly to the match between Chile and Poland, as both Canada and Poland won their first three picks. [trig0n](https://osu.ppy.sh/users/3704228) matched [Aryssiels](https://osu.ppy.sh/users/2389481) insane full-combo on [HR1](https://osu.ppy.sh/beatmapsets/2198791#fruits/4653099) and [Phob](https://osu.ppy.sh/users/6069462) got an impressive one-miss on [HR2](https://osu.ppy.sh/beatmapsets/2198874#fruits/4653271).
```

Comment:
Arrysiel? Is this supposed to be matching like the best score on the map period, or did you put the wrong player here?
```suggestion
This match started very similarly to the match between Chile and Poland, as both Canada and Poland won their first three picks. ::{ flag=PL }:: [trig0n](https://osu.ppy.sh/users/3704228) matched ::{ flag=CL }:: [Aryssiels](https://osu.ppy.sh/users/2389481) insane full-combo on [HR1](https://osu.ppy.sh/beatmapsets/2198791#fruits/4653099), and ::{ flag=PL }:: [Phob](https://osu.ppy.sh/users/6069462) got an impressive one-miss on [HR2](https://osu.ppy.sh/beatmapsets/2198874#fruits/4653271).
```

Reply:
Aryssiel FCd the map the day before, now trig0n FCd today. Might be a bit too ambigious wording